src/provider/constants.ts

```diff
@@ -7,6 +7,11 @@ export const Timeouts = {
      */
     ADD_INTENT_LISTENER: 5000,
 
+    /**
+     * Time for an app to register a listener after opening
+     */
+    ADD_CONTEXT_LISTENER: 5000,
```

Comment:
Set this to the same as Intents, but no objection to changing if people have other opinions.